specification/billing/resource-manager/Microsoft.Billing/stable/2024-04-01/billingSubscription.json

```diff
@@ -0,0 +1,2065 @@
+        },
+        "value": {
+          "description": "The list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingSubscription"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": [
+            "id"
+          ]
+        }
+      }
+    },
+    "BillingSubscriptionMergeRequest": {
+      "description": "Request parameters that are provided to merge the two billing subscriptions.",
+      "type": "object",
+      "properties": {
+        "targetBillingSubscriptionName": {
+          "description": "The ID of the target billing subscription that will be merged with the source subscription provided in the request.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of the source billing subscription that will be merged with the target billing subscription.",
+          "type": "integer"
+        }
+      }
+    },
+    "BillingSubscriptionPatch": {
+      "description": "The billing properties of a subscription.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./types.json#/definitions/PatchResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a(n) BillingSubscription",
+          "$ref": "#/definitions/BillingSubscriptionProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "BillingSubscriptionProperties": {
+      "description": "The billing properties of a subscription.",
+      "type": "object",
+      "properties": {
+        "autoRenew": {
+          "description": "Indicates whether auto renewal is turned on or off for a product.",
+          "enum": [
+            "Off",
+            "On"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutoRenew",
+            "modelAsString": true
+          }
+        },
+        "beneficiaryTenantId": {
+          "description": "The provisioning tenant of the subscription.",
+          "type": "string"
+        },
+        "beneficiary": {
+          "description": "The beneficiary of the billing subscription.",
+          "$ref": "#/definitions/Beneficiary"
+        },
+        "billingFrequency": {
+          "description": "The billing frequency in ISO8601 format of product in the subscription. Example: P1M, P3M, P1Y",
+          "type": "string"
+        },
+        "billingProfileId": {
+          "description": "The fully qualified ID that uniquely identifies a billing profile.",
+          "type": "string"
+        },
+        "billingPolicies": {
+          "type": "object",
+          "description": "Dictionary of billing policies associated with the subscription.",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "maxItems": 50,
+          "readOnly": true
+        },
```

Comment:
I wonder why its modeled with additionalProperties instead of a list.

Reply:
I guess it could be... but this has been there since 2021-10-01 which would make it a breaking change now right?